mods/creative/init.lua

```diff
@@ -46,7 +46,7 @@ end
 
 -- Unlimited node placement
 minetest.register_on_placenode(function(pos, newnode, placer, oldnode, itemstack)
-	return creative.is_enabled_for(placer:get_player_name())
+	return placer and creative.is_enabled_for(placer:get_player_name()) or false
```

Comment:
It does not matter whether true or false is returned, does it?

Reply:
It does matter because if true is returned then the stack is not modified (I think?). But false should be the correct value since non-players do not enter creative mode.